src/frontend/src/lib/services/address.services.ts

```diff
@@ -55,7 +60,9 @@ const loadTokenAddress = async <T extends Address>({
 		const address = await getAddress(identity);
 		addressStore.set({ data: address, certified: true });
 
-		await saveTokenAddressForFutureSignIn({ address, identity, setIdbAddress });
+		if (nonNullish(setIdbAddress)) {
```

Comment:
Should we have a test for this? If a bug is introduced this can potentially have quite an impact on the user experience.

Reply:
I agree with tests, but since there are no tests for these services, I didn't think it necessary.

We should start by having tests.

Reply:
We have E2E tests for the login now no @DenysKarmazynDFINITY ? Do we also test re-entering the app with a valid identity ? i.e. are we testing that when the use returns the modal to load the addresses is not displayed?

Reply:
We have tests for the login and the logout actions but not for re-entering. Would be cool to have it though (we have it listed in backlog).

Reply:
Given this change, if you don't mind, let's then priorize this particular test in the backlog

Reply:
Agreed 👍